test/integration/fixtures.ts

```diff
@@ -0,0 +1,27 @@
+import config from "../../src/config";
+import { Mjolnir } from "../../src/Mjolnir";
+import { makeMjolnir, teardownManagementRoom } from "./mjolnirSetupUtils";
+
+// when mjolnir starts it clobbers the config, which is cached between runs,
```

Comment:
What clobbers the config? Mjölnir or this object?

Reply:
I've tried to clarify this in the file, but for here too, basically Mjolnir the class from Mjolnir.ts expects that config.managementRoom is a roomId but the config also allows you to specify an alias. The tests can only use the alias version as they need a new management room to avoid getting confused with state from previous tests. When mjolnir is setup to start, the config instance (which is included by every file where it is referenced directly and they all share the same instance) is mutated to ensure that this value is a room ID.